src/main/java/ca/xqz/tweetmouth/TweetLoader.java

```diff
@@ -63,9 +63,13 @@ public void loadTweets(List<Tweet> tweets) {
         }
     }
 
-    private void maybeCreateIndex() {
+    private boolean indexExists() {
         IndicesExistsResponse resp = client.admin().indices().exists(new IndicesExistsRequest(index)).actionGet();
-        if (!resp.isExists()) {
+        return resp.isExists();
+    }
+
+    private void maybeCreateIndex() {
+        if (!indexExists()) {
```

Comment:
lolwut
This method should be renamed to `createIndex()` and the `if (!index.exists())` should go in the caller.